atc/db/team_test.go

```diff
@@ -3289,6 +3289,62 @@ var _ = Describe("Team", func() {
 		})
 	})
 
+	Describe("RenamePipeline", func() {
+		It("renames individual pipelines", func() {
+			found, err := defaultTeam.RenamePipeline("default-pipeline", "new-pipeline")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(found).To(BeTrue())
+
+			_, err = defaultPipeline.Reload()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(defaultPipeline.Name()).To(Equal("new-pipeline"))
+		})
+
+		Context("when multiple pipeline instances have the same name", func() {
```

Comment:
Can you add a test for renaming a set of pipelines that have the same name, but one of them doesn't have instance vars?